Source/dom/layers/__tests__/Text.test.js

```diff
@@ -78,3 +78,60 @@ test('should fix the width', () => {
 
   expect(text.fixedWidth).toBe(true)
 })
+
+test('should return the fragments of a text layer', () => {
+  let text = new Text({
+    text: 'blah',
+  })
+  text.adjustToFit()
+
+  let { fragments } = text
+
+  expect(fragments.length).toBe(1)
+  expect(fragments[0].baselineOffset).toBe(3)
+  expect(Number(fragments[0].range.location)).toBe(0)
+  expect(Number(fragments[0].range.length)).toBe(4)
+  expect(fragments[0].rect.toJSON()).toEqual({
+    x: 0,
+    y: 0,
+    width: 10000000,
```

Comment:
This looks like a magical number, not sure what's going on here.

Reply:
Found it! Not sure it ever worked properly...